templates/download/risc-v/index.html

```diff
@@ -1,74 +1,112 @@
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="false"
+                      aria-controls="microchip-polarfire-soc-fpga-icicle-kit-tab"
+                      id="microchip-polarfire-soc-fpga-icicle-kit-tab">Microchip Polarfire SoC FPGA Icicle</button>
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="false"
+                      aria-controls="milk-v-mars-tab"
+                      id="qemu-tab">Milk-V Mars</button>
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="false"
+                      aria-controls="qemu-tab"
+                      id="qemu-tab">QEMU</button>
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="true"
+                      aria-controls="quemu-sifive-image-tab"
+                      id="quemu-sifive-image"
+                      tabindex="-1">SiFive Unmatched</button>
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="false"
+                      aria-controls="sipeed-licheerv-dock-tab"
+                      id="sipeed-licheerv-dock-tab">Sipeed LicheeRV Dock</button>
+            </div>
+            <div class="p-tabs__item">
+              <button class="p-tabs__link"
+                      role="tab"
+                      aria-selected="false"
+                      aria-controls="visionfive-2-board-tab"
+                      id="starfive-visionfive-board">赛昉VisionFive 2</button>
+            </div>
+          </div>
         </div>
       </div>
-      {% include "download/risc-v/tab-1.html" %}
-      {% include "download/risc-v/tab-2.html" %}
-      {% include "download/risc-v/tab-3.html" %}
-      {% include "download/risc-v/tab-4.html" %}
-      {% include "download/risc-v/tab-5.html" %}
-      {% include "download/risc-v/tab-6.html" %}
+      <div class="col-8 col-medium-4">
+        {% include "download/risc-v/tab-1.html" %}
+        {% include "download/risc-v/tab-2.html" %}
+        {% include "download/risc-v/tab-3.html" %}
+        {% include "download/risc-v/tab-4.html" %}
+        {% include "download/risc-v/tab-5.html" %}
+        {% include "download/risc-v/tab-6.html" %}
+        {% include "download/risc-v/tab-7.html" %}
+      </div>
     </div>
-  </div>
-</section>
-
-<script src="/static/js/src/tabbedContent.js"></script>
-
-{% endblock content %}
\ No newline at end of file
+  </section>
+  <script src="/static/js/src/tabbedContent.js"></script>
```

Comment:
Optionally (I think it'd be good learning) would you be able change this to use `versioned_static` for this? e.g. https://github.com/canonical/ubuntu.com/blob/f469d7ec8d1e83207e8221bb1dff42477ffb94ed/templates/ceph/install.html#L277